spdx-license-adder.py

```diff
@@ -0,0 +1,83 @@
+import os
```

Comment:
It turns out that Google has published an open source program named `addlicense` that is already used by the p4lang/p4-constraints repository for this purpose, and I think it would be a good idea to enhance it upstream to recognize the `.p4` file name suffix than it would be to create our own.  I have a fork that does this here: https://github.com/p4lang/addlicense

Reply:
Indeed, I realized that I was too late.